### DISCUSSION TITLE
How to include `neff` in jagstargets summary

### DISCUSSION THREAD

**Discussion by neosanma** [wlandau/jagstargets#30] [Help] [answered]
Hello everyone, 

I was wondering if somebody knew how to output a summary table that includes the neff (effective number of samples) of the jags run when using `tar_jags`. 

Thanks!

**Comment by wlandau** [comment, marked as answer]
In the example at https://docs.ropensci.org/jagstargets/articles/introduction.html, `tar_read(example_summary_x)` has effective sample size in columns `ess_bulk` and `ess_tail`. Both are calculated by `posterior::summarize_draws()`. If you want basic effective sample size instead of `posterior::ess_bulk()` or `posterior::ess_tail()`, you can customize the `summaries` argument of `tar_jags()`, e.g. `summaries = list(ess_basic = ~posterior::ess_basic(.x))`.

**Reply by neosanma** [reply]
Thanks Will!
